tests/WizardTest.php

```diff
@@ -184,4 +192,50 @@ public function testWizardTestSteps(): void
         self::assertEquals($stepBySlug::$slug, $this->secondTestStepClass::$slug);
     }
 
-}
+    public function testWizardAppendSteps(): void
+    {
+        $this->wizard = $this->createPartialMock(Wizard::class, []);
+        $this->wizard->__construct($this->steps);
+
+        $result = $this->wizard->appendStep($this->fourthTestStepClass::class, $this->wizardFourthStepKey);
```

Comment:
Here i don't know we should use php 8.0 feature like '::class'  on object. But if i rewrite this test it shoud not be needed